SwiftVoiceCallKitQuickstart/ViewController.swift

```diff
@@ -193,6 +194,7 @@ class ViewController: UIViewController, PKPushRegistryDelegate, TVONotificationD
         NSLog("callDidConnect:")
         
         self.call = call
+        self.callKitCompletionCallback!(true)
```

Comment:
Minor - you could also clear `self.callKitCompletionCallback!` here and in the failure case.